<launch>
 <!-- Turtlesim Node-->
<!--  <node pkg="turtlebot" type="turtlesim_node" name="sim"/>-->
<!-- <include file="$(find turtlebot3_gazebo)/launch/ipc_serial.launch"/>-->
 <!-- 必须包含joy node -->
 <!-- joy node -->
  <node respawn="true" pkg="joy"
        type="joy_node" name="turtlebot3_joy" >
    <param name="dev" type="string" value="/dev/input/js0" />
    <param name="deadzone" value="0.12" />
  </node>

 <!-- Axes -->
  <param name="axis_linear" value="1" type="int"/>
  <param name="axis_angular" value="0" type="int"/>
<!-- 在此处改最大速度-->
  <param name="scale_linear" value="1.5" type="double"/>
  <param name="scale_angular" value="1.5" type="double"/>
<!-- type是可执行文件-->
  <node pkg="turtlebot3_joy" type="turtlebot3_teleop_joy" name="teleop"/>
</launch>
